server/utils.py

```diff
@@ -77,6 +79,13 @@ def is_safe_redirect_url(request, target):
     return redirect_url.scheme in ('http', 'https') and \
         host_url.netloc == redirect_url.netloc
 
+def str_to_tz(tz_str):
+    if tz_str not in pytz.common_timezones_set:
+        logger.warning("Invalid TZ choice {}".format(tz_str))
+        return None
```

Comment:
This isn't a case we'll hit since form validation ensures timezone is valid. 

The other approach is to type check in the SQLAlchemy converter (and support String (Python) -> String (DB) storage instead of having to do String (HTML) -> Timezone (Python) -> String (DB)